src/fr/yaoiscan/src/YaoiScan.kt

```diff
@@ -0,0 +1,240 @@
+package eu.kanade.tachiyomi.extension.fr.yaoiscan
+
+import eu.kanade.tachiyomi.multisrc.mangareader.MangaReader
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.network.asObservableSuccess
+import eu.kanade.tachiyomi.source.model.Filter
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.util.asJsoup
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
+import okhttp3.Response
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import org.jsoup.select.Evaluator
+import rx.Observable
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class YaoiScan : MangaReader() {
+
+    override val name = "YaoiScan"
+
+    override val lang = "fr"
+
+    override val baseUrl = "https://yaoiscan.fr/"
+
+    override val client = network.cloudflareClient.newBuilder()
+        // .rateLimit(2)
```

Comment:
I have questions for this. Should I fully remove the override or by default this is not a cloudflare client ?

Reply:
[Multisrc](<https://github.com/keiyoushi/extensions-source/blob/f15e50aa0a69bce289fd7d8f01b8620315870e7f/lib-multisrc/mangareader/src/eu/kanade/tachiyomi/multisrc/mangareader/MangaReader.kt#L25>) is already using `cloudflareClient` by default, so you should use `super.client` if you want to override it (`rateLimit` is comented?)